streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -710,7 +704,13 @@ public void apply(final StreamTask task) {
                 maybePunctuate(task);
                 if (task.commitNeeded()) {
                     name = "commit";
+
+                    long beforeCommitMs = time.milliseconds();
+
                     commitOne(task);
+
+                    log.info("{} Committed active task {} in {}ms",
```

Comment:
This is actually a top-level event, that user requested commit (note we have two types of such events: either commit interval has elapsed in which case all tasks are committed; or user called `commit` in which case only this single task is committed). We are using `INFO` for all such top-level events.